guides/plugins/apps/starter/product-translator.md

```diff
@@ -0,0 +1,363 @@
+APP_NAME=product-translator
+APP_ENV=dev
+APP_SECRET=01f17b06402f0a24e6d2b084a6d18a87
+APP_DEBUG=true
+###< symfony/framework-bundle ###
+
+###> doctrine/doctrine-bundle ###
+# Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
+# IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml
+#
+DATABASE_URL=mysql://root:root@127.0.0.1:3306/product_translator?serverVersion=8.0
+###< doctrine/doctrine-bundle ##
+```
+{% endcode %}
+
+You can now start the application with `symfony server:start -v`.
+
+For now, your app server is currently only available locally.
+
+{% hint style="info" %}
+When you are using a local Shopware environment, you can skip to the [next chapter](#creating-the-manifest)
+{% endhint %}
+
+We need to expose your local app server to the internet. The easiest way to achieve that is using a tunneling service like [ngrok](https://ngrok.com/). 
+
+The setup is as simple as calling the following command (after installing ngrok)
+
+```sh
+ngrok http 8000
+```
+
+This will expose your symfony server on a public URL, so the cloud store can communicate with your app.
+
+## Creating the manifest
+
+The `manifest.xml` is the main interface definition between stores and your app server.
+It contains all required information about your app.
+Let's start by filling in all the meta information:
+
+{% code title="release/manifest.xml" %}
+``` xml
+<?xml version="1.0" encoding="UTF-8" ?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/master/src/Core/Framework/App/Manifest/Schema/manifest-1.0.xsd">
+    <meta>
+        <name>product-translator</name>
+        <label>Product translator</label>
+        <label lang="de-DE">Product übersetzer</label>
+        <description>App to translate product descriptions</description>
+        <description lang="de-DE">Product übersetzer</description>
+        <author>shopware AG</author>
+        <copyright>(c) by shopware AG</copyright>
+        <version>0.1.0</version>
+        <license>MIT</license>
+    </meta>
+   </manifest>
+```
+{% endcode %}
+
+{% hint style="warning" %}
+Take care to use the same `<name>` as in the `.env` file, otherwise stores can't install the app!
+{% endhint %}
+
+Next up we will define the `<setup>` part of the manifest. This part describes how the 
+store will connect itself with the app server.
+
+{% code title="release/manifest.xml" %}
+``` xml
+<?xml version="1.0" encoding="UTF-8" ?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/master/src/Core/Framework/App/Manifest/Schema/manifest-1.0.xsd">
+    <meta>
+    <!-- ... -->
+    </meta>
+    <setup>
+        <registrationUrl>http://127.0.0.1:8000/register</registrationUrl>
```

Comment:
Mention usage of ngrok URL here - Cloud development should be assumed as the "standard" case in this guide. Exceptions are always formulated as "When you're using a self-managed ... "